pkg/controller/backup/backup_controller.go

```diff
@@ -53,13 +55,17 @@ func NewController(deps *controller.Dependencies) *Controller {
 	}
 
 	backupInformer := deps.InformerFactory.Pingcap().V1alpha1().Backups()
+	jobInformer := deps.KubeInformerFactory.Batch().V1().Jobs()
 	backupInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc: c.updateBackup,
```

Comment:
why we are enqueueing it but handle it directly

Reply:
To my knowledge, in order to don't block `Informer`'s execution.